01-html/01-koajs.html

```diff
@@ -1,4 +1,671 @@
+        <code>this.assert(this.state.user, 401, 'User not found. Please login!');</code>
+        <p>koa uses <a href="https://github.com/jshttp/http-assert">http-assert</a> for assertions.</p>
+      <h3>ctx.respond</h3>
+        <p>To bypass Koa's built-in response handling, you may explicitly set <em>this.respond = false;</em>. Use this if you want to write to the raw <em>res</em> object instead of letting Koa handle the response for you.</p>
+        <p>Note that using this is <b>not</b> supported by Koa. This may break intended functionality of Koa middleware and Koa itself. Using this property is considered a hack and is only a convenience to those wishing to use traditional <em>fn(req, res)</em> functions and middleware within Koa.</p>
+      <h3>Request aliases</h3>
+        <p>the following accessors and alias <a href="http://koajs.com/#request">Request</a>:</p>
+        <ul>
+          <li>ctx.header</li>
+          <li>ctx.headers</li>
+          <li>ctx.method</li>
+          <li>ctx.method=</li>
+          <li>ctx.url</li>
+          <li>ctx.url=</li>
+          <li>ctx.originalUrl</li>
+          <li>ctx.origin</li>
+          <li>ctx.href</li>
+          <li>ctx.path</li>
+          <li>ctx.path=</li>
+          <li>ctx.query</li>
+          <li>ctx.query=</li>
+          <li>ctx.querystring</li>
+          <li>ctx.querystring=</li>
+          <li>ctx.host</li>
+          <li>ctx.hostname</li>
+          <li>ctx.fresh</li>
+          <li>ctx.stale</li>
+          <li>ctx.socket</li>
+          <li>ctx.protocol</li>
+          <li>ctx.secure</li>
+          <li>ctx.ip</li>
+          <li>ctx.ips</li>
+          <li>ctx.subdomains</li>
+          <li>ctx.is()</li>
+          <li>ctx.accepts()</li>
+          <li>ctx.acceptsEncodings()</li>
+          <li>ctx.acceptsCharsets()</li>
+          <li>ctx.acceptsLanguages()</li>
+          <li>ctx.get()</li>
+        </ul>
+      <h3>Response aliases</h3>
+        <p>The following accessors and alias <a href="http://koajs.com/#response">Response</a> equivalents:</p>
+        <ul>
+          <li>ctx.body</li>
+          <li>ctx.body=</li>
+          <li>ctx.status</li>
+          <li>ctx.status=</li>
+          <li>ctx.message</li>
+          <li>ctx.message=</li>
+          <li>ctx.length=</li>
+          <li>ctx.length</li>
+          <li>ctx.type=</li>
+          <li>ctx.type</li>
+          <li>ctx.headerSent</li>
+          <li>ctx.redirect()</li>
+          <li>ctx.attachment()</li>
+          <li>ctx.set()</li>
+          <li>ctx.append()</li>
+          <li>ctx.remove()</li>
+          <li>ctx.lastModified=</li>
+          <li>ctx.etag=</li>
+        </ul>
+    <h2>Request</h2>
+      <p>A Koa <em>Request</em> object is an abstraction on top of node's vanilla request object, providing additional functionality that is useful for every day HTTP server development.</p>
+      <h3>API</h3>
+      <h3>request.header</h3>
```

Comment:
Note here that you are using the same tag for `request.header` as you are for `API`, when the former is a sub-item of the latter. 

This section might be best served by implementing a *definition list*, such as:

'''
<dl> 
  <dt>request.header</dt>
  <dd>Request header object</dd>
  <dt>request.headers</dt>
...
'''